aspnetcore/mvc/views/view-compilation/sample/MvcRazorCompileOnPublish.csproj

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
```

Comment:
You could remove this. Shouldn't need it. Similarly the scaffolding tool. Might be nice to have a minimal sample